### DISCUSSION TITLE
Overcome hyprland wrapper

### DISCUSSION THREAD

**Discussion by SimoneRiccardi** [hyprwm/Hyprland#1690] [General]
use a [wrapper](http://wiki.hyprland.org/Getting-Started/Master-Tutorial/#launching-hyprland-part-1) to set the necessary env variables and exec Hyprland is more inconvenient that have the possibility to simply write ```Hyprland``` in your terminal or use the default ```/usr/share/wayland-sessions/``` file (that launch the command ```exec Hyprland```)  and have Hyprland itself that execute a script 

i propose an ```hyprlandrc``` in the same folder of ```hyprland.conf```  that'll be a script executed before loading ```hyprland.conf``` and the user will write the enviroment variables there.

**Comment by vaxerski** [comment]
an issue on the tracker for this would be better
